src/JavascriptErrorContext.php

```
@@ -0,0 +1,129 @@
+ */
+class JavascriptErrorContext extends MinkContext implements Context {
+
+  /**
+   * Errors registry keyed by path.
+   */
+  private array $jsErrorRegistry = [];
+
+  /**
+   * Step patterns.
+   */
+  private array $stepFilters = [
```

PHPStan (2.1.15)

Property DrevOps\BehatSteps\JavascriptErrorContext::$stepFilters type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)